pyrevitlib/pyrevit/forms/__init__.py

```diff
@@ -1016,6 +1016,16 @@ def _get_active_ctx(self):
         else:
             return self._context
 
+    def _get_all_ctx(self):
+        """Get all context items across all groups"""
+        if isinstance(self._context, dict):
+            all_items = []
```

Comment:
Micro optimization here: since this list is used for a list comprehension by the caller method, we can save some memory by turning it into a generator by `yield`ing the items directly